modules/cpmso-yaml-sample-cr.adoc

```diff
@@ -54,11 +54,15 @@ $ oc get -o jsonpath='{.status.infrastructureName}{"\n"}' infrastructure cluster
 +
 [IMPORTANT]
 ====
-Before you activate the Operator, you must ensure that the `ControlPlaneMachineSet` CR configuration is correct for your cluster requirements.
-//For more information about activating the Control Plane Machine Set Operator, see "Getting started with the Control Plane Machine Set Operator".
+Before you activate the Operator, you must ensure that the `ControlPlaneMachineSet` CR configuration is correct for your cluster requirements. For more information about activating the Control Plane Machine Set Operator, see "Getting started with the Control Plane Machine Set Operator".
```

Comment:
In the preview, the left hand navigation lists "Getting started". Consider updating it to "Getting started with the Control Plane Machine Set Operator" for ease of search and to comply with other sibling topics that mention the full Operator name.